SohamGaikwad/Beginner/ContactApp/app/src/main/java/com/example/contactapp/ContactDetailActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.example.contactapp;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.io.File;
+
+public class ContactDetailActivity extends AppCompatActivity {
+    private String contactName, contactNumber, contactEmail;
+    private TextView contactTV, nameTV, emailTV;
+    private ImageView contactIV;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
Fields can be converted to local variables in onCreate(). Casting 'findViewById(...)' to 'ImageView' is redundant. Please take care of such issues. Clean code & remove unused imports.

Reply:
Fixed in ad34fcde338dffceb4d0ffa797d1c1bfc738641e.